static/js/services/alertify.js

```diff
@@ -89,6 +89,15 @@ define(["angular"], function(angular) {
 			});
 		};
 
+		var customIDs = {
```

Comment:
I still do not get why this needs to be defined here. Define it where the dialog is called (in this case in uicontroller).